packages/atomic/src/components/commerce/atomic-commerce-query-error/atomic-commerce-query-error.spec.ts

```diff
@@ -0,0 +1,328 @@
+    });
+
+    it('should display the correct title', () => {
+      expect(locators.title).toHaveTextContent(
+        'Your organization searchuisamples has no available content.'
+      );
+    });
+
+    it('should display the correct description', () => {
+      expect(locators.description).toHaveTextContent(
+        'Add content sources or wait for your newly created sources to finish indexing.'
+      );
+    });
+
+    it('should not display the show more info button', () => {
+      expect(locators.showMoreInfoButton).not.toBeInTheDocument();
+    });
+
+    it('should display the doc link', () => {
+      expect(locators.docLink).toBeInTheDocument();
+    });
+  });
+
+  describe('when the error type is "InvalidTokenException"', () => {
+    let locators: any;
+
+    beforeEach(async () => {
+      const mockError = {
+        type: 'InvalidTokenException',
+        statusCode: 401,
+        message: 'Invalid token',
+      };
+      ({locators} = await renderQueryError({error: mockError}));
+    });
+
+    it('should display the correct icon', () => {
+      expect(locators.icon).toHaveAttribute('icon', CannotAccess);
+    });
+
+    it('should display the correct title', () => {
+      expect(locators.title).toHaveTextContent(
+        'Your organization searchuisamples cannot be accessed.'
+      );
+    });
+
+    it('should display the correct description', () => {
+      expect(locators.description).toHaveTextContent(
+        'Ensure that the token is valid.'
+      );
+    });
+
+    it('should not display the show more info button', () => {
+      expect(locators.showMoreInfoButton).not.toBeInTheDocument();
+    });
+
+    it('should display the doc link', () => {
+      expect(locators.docLink).toBeInTheDocument();
+    });
+  });
+
+  describe('when the error type is "OrganizationIsPausedException"', () => {
+    let locators: any;
+
+    beforeEach(async () => {
+      const mockError = {
+        type: 'OrganizationIsPausedException',
+        statusCode: 403,
+        message: 'Organization is paused',
+      };
+      ({locators} = await renderQueryError({error: mockError}));
+    });
+
+    it('should display the correct icon', () => {
+      expect(locators.icon).toHaveAttribute('icon', SearchInactive);
+    });
+
+    it('should display the correct title', () => {
+      expect(locators.title).toHaveTextContent(
+        'Your organization searchuisamples is paused due to inactivity and search is currently unavailable.'
+      );
+    });
+
+    it('should display the correct description', () => {
+      expect(locators.description).toHaveTextContent(
+        'Your organization is resuming and will be available shortly.'
+      );
+    });
+
+    it('should not display the show more info button', () => {
+      expect(locators.showMoreInfoButton).not.toBeInTheDocument();
+    });
+
+    it('should display the doc link', () => {
+      expect(locators.docLink).toBeInTheDocument();
+    });
+  });
+});
```

Comment:
Maybe a test to ensure getAriaMessageFromErrorType is called with the appropriate args... the case with `hasError ===true` is not covered